pages/apis/rest_api/clusters.md

```diff
@@ -0,0 +1,86 @@
+# Clusters API
+
+## Cluster data model
+
```

Comment:
```suggestion
A [cluster](/docs/agent/clusters) is a new way of managing your Buildkite agents. They allow teams to self-manage their agent pools, let admins create isolated sets of agents and pipelines within the one Buildkite organization, and help to make agents and queues more discoverable across your organization.

```

Reply:
Great minds etc etc. I was thinking this missed a bit of context, too. Since this is reference information, I think it probably doesn't need to add as much context about how/why you're using clusters. It just needs to clarify what you're looking at. Thinking:

```suggestion
A [cluster](/docs/agent/clusters) is an isolated set of agents and pipelines within an organization.

```